### PR TITLE
Fix a couple of bad Markdown links

### PR DESCRIPTION
**Pull request by chris-morgan**

### CODE REVIEW

File: src/libstd/fs.rs [hunk 1]
```diff
@@ -81,7 +81,8 @@ use time::SystemTime;
 /// # }
 /// ```
 ///
-/// [`BufReader`]: ../io/struct.BufReader.html
+//! [`Read`]: ../io/trait.Read.html
```
Comment (TimNN) [inline src/libstd/fs.rs]: The exclamation mark looks like a typo.

Comment (chris-morgan) [inline src/libstd/fs.rs]: Fixed.

**Comment by chris-morgan** [comment]
r?

**Pull request merged**